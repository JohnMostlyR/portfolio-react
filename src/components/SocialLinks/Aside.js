import styled from 'styled-components';

const Aside = styled.aside`
  float: right;
  padding: 0 1em 0 4em;
  background-image: linear-gradient(45deg, transparent 23%, #fff 23%);
  color: #575756;
`;

export default Aside;
