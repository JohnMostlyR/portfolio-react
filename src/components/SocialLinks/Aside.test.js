import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Aside from './Aside';

describe('Aside', () => {
  it('Should render and match the snapshot', () => {
    const tree = renderer.create(<Aside/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});