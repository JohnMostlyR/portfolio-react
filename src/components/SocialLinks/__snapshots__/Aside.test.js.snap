// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aside Should render and match the snapshot 1`] = `
.c0 {
  float: right;
  padding: 0 1em 0 4em;
  background-image: linear-gradient(45deg,transparent 23%,#fff 23%);
  color: #575756;
}

<aside
  className="c0"
/>
`;
